src/modules/endpoint/__tests__/DatabaseConnector.spec.js

```diff
@@ -102,38 +103,36 @@ describe('DatabaseConnector', () => {
     it('should return a moment that matches the one that was stored', async () => {
       const context = new DatabaseContext('tcc', 'de')
 
-      const dateExpected = moment.tz('20110205', 'UTC')
+      const dateExpected = moment('2011-05-04T00:00:00.000Z')
 
       await databaseConnector.storeLastUpdate(dateExpected, context)
-      const dateReceived = await databaseConnector.loadLastUpdate(context)
-
-      expect(dateExpected.isSame(dateReceived)).toBe(true)
+      expect(dateExpected).toStrictEqual(await databaseConnector.loadLastUpdate(context))
```

Comment:
Shouldn't this be:
```suggestion
      expect(dateExpected.isSame(await databaseConnector.loadLastUpdate(context)).toBe(true)
```

I don't get why you need a strict equal here (and why the test doesn't fail, since the moment instances are not the same, I guess?)

Reply:
It just assures all properties are the same, kind of like equal https://jestjs.io/docs/en/expect.html#tostrictequalvalue. You're thinking of `toBe` which fails if the instances are not the same.